### ISSUE TITLE
Gym info seems invalid?

### ISSUE THREAD

**Issue by darkhax** [Necrobot-Private/NecroBot#1908]
### Expected Behavior:
The gym waypoint should have useful info. Maybe you could add some other info as well?

### Actual Behavior:
Some fields of the gym display, like CP and Gym Points is 0. 

### Detailed Steps to Reproduce:
1. Find a gym in the map view, it's the 3rd one and looks like a pokeball.
2. Click on it.
3. The CP and Gym Points should be 0. 

### Other Information
(Attach zip files if needed.)

![image](https://user-images.githubusercontent.com/2250798/28496462-40b7546e-6f29-11e7-8495-44137362bc76.png)

Version: 1.0.0.185
Windows Version: 10
What is affected (catch, transfer, favorite, GPX, evolve, recycle, snipe, human walk snipe, pokestops, powerup/levelup, other): Map

**Comment by blackuGT** [comment]
NecroBot2 v1.0.0.186 - all gyms :
Gym points: 0 | Lvl: 1

+ Bot doesn't attack, train, deploy pokemons in gyms.

**Comment by Furtif** [comment]
@blackuGT normal procedure into GUI gym info is not ok because it as changed from getgymdetails to fymgetinfo into protos and all code need revised.

**Comment by blackuGT** [comment]
> all code need revised

When will the update come out with revised gym support?

**Comment by Furtif** [comment]
@blackuGT I do not know, it depends on the pogoprotos functions available, for the moment the developers do not want to integrate the modes of attak raid among others

**Comment by blackuGT** [comment]
@Furtif well, There is nothing else to wait for "next" Gym update, i think this thread has been depleted.

**Comment by Furtif** [comment]
@blackuGT  Yes there is, there are always new things and others fix, and who knows this is decided otherwise ...

**Comment by dimitris63** [comment]
I am sorry for not understanding. Can we have gyms enabled or not? If we can enable then is there any special configs.? Also same question about raids.

**Comment by blackuGT** [comment]
@dimitris63 Gyms are not supported.

**Comment by blackuGT** [comment]
#InvalidResponseException#  (GYM errors) :

PokemonGo.RocketAPI.Exceptions.InvalidResponseException: Error with API request type: GymGetInfo

20-25 errors / day. Error cause bot restart.

**Comment by dimitris63** [comment]
Do you know when they will be supported
